### PR TITLE
fixed bug in riemann tensor

### LINKED ISSUE #578: Formula for calculation of Reimann Tensor from Christoffel Symbols is incorrect
🐞 **Problem**

See title and discussion in PR #573.

🎯 **Goal**

To fix inaccuracies in the package.

💡 **Possible solutions**

Change the formula, according to the discussion in PR #573.

📋  **Steps to solve the problem**

See PR #577.

### PR DESCRIPTION
**Pull request by aweinr4**
Fixes #578

### CODE REVIEW

**Comment by aweinr4** [comment]
after far too many attempts I believe this fixes the bug and updates the changelog and has only one commit

File: src/einsteinpy/symbolic/riemann.py [hunk 1]
```diff
@@ -55,7 +55,11 @@ def __init__(
     @classmethod
     def from_christoffels(cls, chris, parent_metric=None):
         """
-        Get Riemann Tensor calculated from a Christoffel Symbols
+        Get Riemann Tensor calculated from Christoffel Symbols.
+        Reimann Tensor is given as:
+        .. math::
```
Comment (JeS24) [inline src/einsteinpy/symbolic/riemann.py:60]: @aweinr4 Insert a blank line here. That should fix the doc-build.

File: CHANGELOG [hunk 2]
```diff
@@ -44,6 +45,7 @@ CHANGES
 - [#548]: Fixed #535, Moved to a pure python geodesic solver, Julia dependency removed
 - [#548]: Minor edits to documentation for geodesic and plotting.geodesic modules
 - [#571]: Fixed #570, Updated Master to Main
+- [#573]: Fixed bug in Riemann Tensor and added equation to docstring
```
Comment (JeS24) [inline CHANGELOG:48]: Just "Fixed bug in Riemann Tensor calculation from Christoffels" would do.

**Review by JeS24 — COMMENTED** [review]
You can make another commit. 2 is a perfectly fine number of commits for this PR.

File: src/einsteinpy/symbolic/riemann.py (hunk 1 above)
Comment (JeS24) [reply src/einsteinpy/symbolic/riemann.py:60]: I mean, L60.

**Comment by aweinr4** [comment]
I looked at the circleci fail it keeps saying exit code 1 without much info. Not sure how to fix

**Comment by JeS24** [comment]
> I looked at the circleci fail it keeps saying exit code 1 without much info. Not sure how to fix

Hmm. Let me take a look.

**Comment by JeS24** [comment]
@aweinr4 This seems to be the issue. Try fixing it and pushing.

> Hello @aweinr4! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:
> 
> * In the file [`src/einsteinpy/symbolic/riemann.py`](https://github.com/einsteinpy/einsteinpy/blob/5b75d93bc341fcc2ef616fe9358f016599a52a20/src/einsteinpy/symbolic/riemann.py):
> 
> > [Line 60:1](https://github.com/einsteinpy/einsteinpy/blob/5b75d93bc341fcc2ef616fe9358f016599a52a20/src/einsteinpy/symbolic/riemann.py#L60): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> 
> ##### Comment last updated at 2021-02-19 14:26:14 UTC


Also, for code formatting (and to avoid code format issues), you can use `tox`. `pip install tox` and run `tox -e reformat` anywhere inside the `einsteinpy` directory.

**Comment by JeS24** [comment]
Thanks @aweinr4. Merging now.

**Pull request merged**